### DISCUSSION TITLE
Mastodon blank page

### DISCUSSION THREAD

**Discussion by theawes0megamer** [mastodon/mastodon#26242] [Q&A] [answered]
when i visit my instance, i get a blank page. (console error 500)
when i visit it on the machine (ubuntu), i look in the console and am getting a 403 error.

also, when i run journalctl -u mastodon-web.service i get a blocked host: alton.cloud

any help here? thanks!
instance: https://social.alton.cloud

**Comment by theawes0megamer** [comment]
also, the favicon loads

**Comment by theawes0megamer** [comment, marked as answer]
Hey guys, I solved this by re-installing it on a new machine. Also mastodon really needs to add usermod -aG mastodon www-data and vice versa to the documentation. That had me stumped.

**Reply by andypiper** [reply]
Can you open this as an issue on the mastodon/documentation repository along with details of where this command should be added, thank you.

**Reply by theawes0megamer** [reply]
> Can you open this as an issue on the mastodon/documentation repository along with details of where this command should be added, thank you.

Yes, I will
